models/sharky/solana/sharky_solana_events.sql

```diff
@@ -0,0 +1,109 @@
+{{ config(
+    alias = 'events',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'tx_hash', 'id'],
+    post_hook='{{ expose_spells(\'["solana"]\',
+                                    "project",
+                                    "sharky",
+                                    \'["ennnas", "hosuke"]\') }}'
+    )
+}}
+
+{%- set project_start_date = '2022-12-30' %} -- TODO change back to '2022-04-14'
```

Comment:
Model building success in 28 mins for 1 month's data.
https://github.com/duneanalytics/spellbook/actions/runs/4054832777/jobs/6977163508#step:9:13

[Sample sharky solana events](https://dune.com/queries/1948433?d=1)